pkg/queue/manager.go

```diff
@@ -459,10 +459,10 @@ func (m *Manager) Heads(ctx context.Context) []workload.Info {
 func (m *Manager) Dump() map[string]sets.String {
 	m.Lock()
 	defer m.Unlock()
-	if len(m.localQueues) == 0 {
+	if len(m.clusterQueues) == 0 {
```

Comment:
wow, I guess the tests always created the same number of cluster and local queues :)